substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/NativeImageAgentJNIHandleSet.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2019, 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.oracle.svm.agent;
+
+import static com.oracle.svm.jni.JNIObjectHandles.nullHandle;
+
+import com.oracle.svm.agent.core.JNIHandleSetBase;
+import com.oracle.svm.jni.nativeapi.JNIEnvironment;
+import com.oracle.svm.jni.nativeapi.JNIMethodId;
+import com.oracle.svm.jni.nativeapi.JNIObjectHandle;
+
+public class NativeImageAgentJNIHandleSet extends JNIHandleSetBase {
+
+    JNIMethodId javaLangClassForName3;
+    JNIMethodId javaLangReflectMemberGetName;
+    JNIMethodId javaLangReflectMemberGetDeclaringClass;
+    JNIMethodId javaUtilEnumerationHasMoreElements;
+    JNIMethodId javaUtilMissingResourceExceptionCtor3;
+    JNIObjectHandle javaLangClassLoader;
+    public JNIObjectHandle javaLangSecurityException;
+    public JNIObjectHandle javaLangNoClassDefFoundError;
+    public JNIObjectHandle javaLangNoSuchMethodError;
+    JNIObjectHandle javaLangNoSuchMethodException;
+    public JNIObjectHandle javaLangNoSuchFieldError;
+    JNIObjectHandle javaLangNoSuchFieldException;
+    JNIObjectHandle javaLangClassNotFoundException;
+    JNIObjectHandle javaLangRuntimeException;
+    JNIObjectHandle javaUtilMissingResourceException;
```

Comment:
Why are these no longer `final`?

Reply:
This might've been lost when I was tweaking the modifiers. I have re-added final.